pode.build.ps1

```diff
@@ -407,7 +408,7 @@ Task DocsHelpBuild DocsDeps, {
                 $line = $_
 
                 while ($line -imatch '\[`(?<name>[a-z]+\-pode[a-z]+)`\](?<char>([^(]|$))') {
-                    $updated = $true
+                    $used:updated = $true
```

Comment:
What's `$used:`?

Reply:
I think I mean to use $using but has no sense :)